package-lock.json

```diff
@@ -699,9 +699,9 @@
 			"integrity": "sha1-rFcX8w8++jYXsATo/0+GC5x9TSA="
 		},
 		"antlr4ts-cli": {
-			"version": "0.4.0-canary.37504efa",
```

Comment:
How come this changed?

Reply:
I ran an npm i to remove any changes to my package.json from the management place API  PR .